Main.hs

```diff
@@ -225,7 +230,7 @@ compileTerm builtins v =
     Var x es   -> (`app` es) . Hs.Var () . Hs.UnQual () . hsName =<< showTCM (Var x [])
     Def f es   -> (`app` es) . Hs.Var () =<< hsQName builtins f
     Con h i es -> (`app` es) . Hs.Con () =<< hsQName builtins (conName h)
-    Lit (LitNat n) -> return $ Hs.Lit () $ Hs.Int () n (show n)
+    Lit (LitNat _ n) -> return $ Hs.Lit () $ Hs.Int () n (show n)
```

Comment:
This is a change since `2.6.1`. Probably makes most sense to add a `< 2.6.2` constraint to the cabal file.

Reply:
ok, I pushed this cabal file change straight to master.